cmd/bucket-handlers.go

```diff
@@ -292,6 +294,16 @@ func (api objectAPIHandlers) ListBucketsHandler(w http.ResponseWriter, r *http.R
 			writeErrorResponse(ctx, w, toAPIError(ctx, err), r.URL, guessIsBrowserReq(r))
 			return
 		}
+
+		for i := range bucketsInfo {
+			configFile := path.Join(bucketConfigPrefix, bucketsInfo[i].Name, bucketCreatedTimeFile)
+			if configData, err := readConfig(ctx, objectAPI, configFile); err == nil {
```

Comment:
Let's make this structured, so we can modify the data later.

```
type bucketMetaData struct {
	Created time.Time
}
```

Since we are doing new [metadata storage](https://github.com/minio/minio/pull/9377) for objects, let's do it in a similar way for buckets.

You can check out how it is serialized by following this: https://github.com/minio/minio/pull/9377/files#diff-a80127ea76a22ee96c3d31240f99b42bR14

Reply:
I see you are also responsible for  `bucketObjectLockEnabledConfigFile` which would have been nice to have in the same file as well. ;)